### PR TITLE
DP-976 funnel icons

### PR DESCRIPTION
**Pull request by gender404**
New Funnel Icons:
https://jira.state.ma.us/browse/DP-691

1. Added new svg files to svg-icons folder
2. Generated new twig files into 05-icons patterns
3. Created new documentation files for each icon
4. Updated existing patterns to use the new icons
5. Removed the old patterns and svg files

### CODE REVIEW

**Comment by gleroux02** [comment]
@legostud I'm getting a gulp error now:
```
patternlab' errored after 2.38 s
[14:28:43] Error: task completion callback called too many times
    at finish (/Users/leroux/workspace/mayflower/styleguide/node_modules/orchestrator/lib/runTask.js:15:10)
    at /Users/leroux/workspace/mayflower/styleguide/node_modules/orchestrator/lib/runTask.js:52:4
    at f (/Users/leroux/workspace/mayflower/styleguide/node_modules/end-of-stream/node_modules/once/once.js:17:25)
    at GulpRunner.onend (/Users/leroux/workspace/mayflower/styleguide/node_modules/end-of-stream/index.js:31:18)
    at emitNone (events.js:85:20)
    at GulpRunner.emit (events.js:179:7)
    at endReadableNT (_stream_readable.js:913:12)
    at _combinedTickCallback (node.js:377:13)
    at process._tickCallback (node.js:401:11)
[14:28:43] Error in plugin 'run-sequence(patternlab)'
Message:
    task completion callback called too many times
Stack:
Error: task completion callback called too many times
    at finish (/Users/leroux/workspace/mayflower/styleguide/node_modules/orchestrator/lib/runTask.js:15:10)
    at /Users/leroux/workspace/mayflower/styleguide/node_modules/orchestrator/lib/runTask.js:52:4
    at f (/Users/leroux/workspace/mayflower/styleguide/node_modules/end-of-stream/node_modules/once/once.js:17:25)
    at GulpRunner.onend (/Users/leroux/workspace/mayflower/styleguide/node_modules/end-of-stream/index.js:31:18)
    at emitNone (events.js:85:20)
    at GulpRunner.emit (events.js:179:7)
    at endReadableNT (_stream_readable.js:913:12)
    at _combinedTickCallback (node.js:377:13)
    at process._tickCallback (node.js:401:11)
```

**Comment by gender404** [comment]
Looks like I need to do a bit more work on the Gulp task.  I've reverted it back to what it was.

File: styleguide/source/_patterns/01-atoms/05-icons/svg-lg-apple.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Apple Icon
```
Comment (gleroux02) [inline styleguide/source/_patterns/01-atoms/05-icons/svg-lg-apple.md]: I'm not able to see the documentation for the new files added. I'm not sure if it has to do with this ^M that is showing in github but maybe patternlab is choking on that?

**Review by gleroux02 — CHANGES_REQUESTED** [review]

**Comment by gender404** [comment]
I converted the line endings to be unix which appears to have removed the weird character in Github.  Not sure if it will fix your issue with documentation not showing up.  The documentation is visible on my local PC.

**Comment by gender404** [comment]
I'm going to merge this pull request and see if the documentation shows up in the dev environment.  If not, I'll create an issue in Github to see if we can find a solution.

**Pull request merged**